node_modules/jest-canvas-mock/types/index.d.ts:1:54 - error TS1183: An implementation cannot be declared in ambient contexts.

I have this lint error being thrown by the typescript:
`node_modules/jest-canvas-mock/types/index.d.ts:1:54 - error TS1183: An implementation cannot be declared in ambient contexts.`

I have this when I am raising a PR in which I am upgrading below packages, will any of the below package bring this issue for jest-canvas-mock? Since it was not thrown before this PR

"immer": "^9.0.19" -> "immer": "^9.0.21"
"react-virtualized-auto-sizer": "^1.0.7" -> "react-virtualized-auto-sizer": "^1.0.9"
"webpack": "^5.76.3" -> "webpack": "^5.77.0"
"@cypress/code-coverage": "^3.10.0" -> "@cypress/code-coverage": "^3.10.1"
"@types/lodash": "^4.14.191" -> "@types/lodash": "^4.14.192"
"@types/node": "^14.18.40" -> "@types/node": "^14.18.42"
 "@types/react": "^16.14.35" -> "@types/react": "^16.14.37"

--- a/README.md
+++ b/README.md
@@ -195,6 +195,7 @@ canvas.toDataURL.mockReturnValueOnce(
 - [@lekha](https://github.com/lekha)
 - [@yonatankra](https://github.com/yonatankra)
 - [@LitoMore](https://github.com/LitoMore)
+- [@hrd543](https://github.com/hrd543)
 
 ## License
 

--- a/types/index.d.ts
+++ b/types/index.d.ts
@@ -1,4 +1,4 @@
-export function setupJestCanvasMock(window?: Window) {}
+export function setupJestCanvasMock(window?: Window): void
 
 export interface CanvasRenderingContext2DEvent {
   /**

